### ISSUE TITLE
kvazaar.exe was built despite standalone=2

### ISSUE THREAD

**Issue by AnryCryman** [jb-alvarado/media-autobuild_suite#579]
Despite the fact that I opted NOT to build standalone libraries, kvazaar.exe was built.

![kvazaar](https://user-images.githubusercontent.com/12110501/28501929-5e0618c2-6fef-11e7-8833-c8d07512371d.jpg)

[media-autobuild_suite.ini.txt](https://github.com/jb-alvarado/media-autobuild_suite/files/1168312/media-autobuild_suite.ini.txt)
[compile.log.txt](https://github.com/jb-alvarado/media-autobuild_suite/files/1168313/compile.log.txt)

**Comment by wiiaboo** [comment]
> other265=1

Don't open another issue on the same topic as one you yourself opened before just to ask a question.

**Comment by AnryCryman** [comment]
That's not a question, that's an issue. I opted to build kvazaar, it was included in FFmpeg, yet I did NOT opt to build standalone libraries. In this case, the only exe-files that I get after the script is done, should be _ffmpeg.exe_, _ffplay.exe_ and _ffprobe.exe_.

**Comment by wiiaboo** [comment]
You said in the .ini that you wanted to build other H.265 encoders. Kvazaar is the only other H.265 besides x265, so I don't know wtf you want.

**Comment by AnryCryman** [comment]
> Kvazaar is the only other H.265 encoder besides x265, so I don't know wtf you want.

I don't want the script to produce kvazaar.exe file when I specify standalone=2.

```
echo. Build standalone binaries for libraries included in FFmpeg?
echo. eg. Compile opusenc.exe if --enable-libopus
echo. 1 = Yes
echo. 2 = No
```

I got my FFmpeg built with --enable-libkvazaar. I don't want kvazaar.exe to be compiled, and it should not be since I chose NOT to build standalone binaries for libraries included in FFmpeg.

**Comment by wiiaboo** [comment]
Then don't use other265=1.
